### ISSUE TITLE
Frostwing Chimaera unusual behaviour

### ISSUE THREAD

**Issue by Braniacz** [dalaranwow/dalaran-wow#5012]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
While questing in Howling Fjord (The Isle of Spears) I encountered mobs "Frostwing Chimaera" that just spin around them selfs. Same problem seems to be with Primordial Drake mob.
**Expected behaviour**: Tell us what should happen instead
Mobs should fly around in designated path.
**Steps to reproduce the problem**:

1. Got to Howling Fjord (The Isle of Spears), Sholazar Basin
2. Find mob Frostwing Chimaera, Primordial Drake
3. Watch spidget finner action

**Include proofs for this behaviour**
I dont think there needs to be any proof since its quite obvious mobs dont move as they should.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Frostwing Chimeara=24673
Primordial Drake=28378

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello and thank you for the report @Braniacz,
I can't confirm the issue on PTR, Frostwing Chimaera looks good to me.
![image](https://user-images.githubusercontent.com/6797611/32619162-8a95f414-c582-11e7-889f-fb2b9e3e3bb5.png)

And the Primordial Drakes will be fixed with #4363

**Comment by Lothloryen** [comment]
@TeridanDW It was related to #5011 , hotfixed it already.
